nginx-sys/build/wrapper.h

```diff
@@ -1,8 +1,23 @@
-#include <ngx_http.h>
-#include <ngx_conf_file.h>
 #include <ngx_config.h>
 #include <ngx_core.h>
 
+/* __has_include was a compiler-specific extension until C23,
+ * but it's safe to assume that bindgen supports it via libclang.
+ */
+#if defined(__has_include)
```

Comment:
Do we need to check for `defined(__has_include)` then?
and why `#include <ngx_stream.h>` is missing if __has_include is absent?